plugins/janus_audiobridge.c

```diff
@@ -2777,6 +2787,28 @@ void janus_audiobridge_incoming_rtp(janus_plugin_session *handle, int video, cha
 		pkt->silence = FALSE;
 		pkt->length = 0;
 
+		//First check if probation period
```

Comment:
Please change all comments that use `//` to `/*   */` instead. Just writing this here, please fix all occurrences.

Reply:
done